pandas/core/arrays/base.py

```diff
@@ -1081,6 +1081,19 @@ def _formatter(self, boxed: bool = False) -> Callable[[Any], Optional[str]]:
     # Reshaping
     # ------------------------------------------------------------------------
 
+    def transpose(self, *axes):
+        """
+        Return a transposed view on this array.
+
+        Because ExtensionArrays are always 1D, this is a no-op.  It is included
```

Comment:
does this method get listed in the api?

Reply:
will add to the class-level docstring

Reply:
ok

Reply:
added it to the docstring and got docbuild trouble.  im OK with skipping it since its a no-op.

Reply:
pls add to list then, this is public API